### PR TITLE
Add a new private package the cfp-template so it can run as a standalone project

### PR DESCRIPTION
**Pull request by janechu**
# Pull Request

## 📖 Description

This change copies the files from the `@microsoft/cfp-template` into a new private package `@microsoft/cfp-template-files`.

This is the first step in de-coupling the logic for a standalone project and the template providing project.

### 🎫 Issues

Begins work on #121 

## 👩‍💻 Reviewer Notes

These files are the same as the ones in the `./packages/cfp-template/template/` folder.

### General

- [ ] I have added tests for my changes.
- [x] I have tested my changes.
- [ ] I have updated the project documentation to reflect my changes.

## ⏭ Next Steps

- Create a `@microsoft/fast-cli` export for reading a project as a template and exporting initialization logic

### CODE REVIEW

File: packages/cfp-template-files/src/components/index.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import { welcome } from "./welcome/define.js";
+
+export default [
+    welcome()
```
Comment (EisenbergEffect) [inline packages/cfp-template-files/src/components/index.ts:4]: This doesn't seem correct. With the new `compose` API, this isn''t a function.

File: packages/cfp-template-files/src/components/welcome/welcome.definition.ts [hunk 2]
```diff
@@ -0,0 +1,8 @@
+import { template } from "./welcome.template.js";
+import { styles } from "./welcome.styles.js";
+
+export const definition = {
```
Comment (EisenbergEffect) [inline packages/cfp-template-files/src/components/welcome/welcome.definition.ts:4]: This all looks like out-of-date foundation code...

**Review by EisenbergEffect — CHANGES_REQUESTED** [review]
Is it possible to update this to the latest fast-element and fast-foundation?

**Comment by janechu** [comment]
> Is it possible to update this to the latest fast-element and fast-foundation?

I'm in the middle of refactoring to solve a pathing issue, I can make that the next body of work.

**Review by EisenbergEffect — APPROVED** [review]

**Pull request merged**